packages/workbox-cache-expiration/src/lib/cache-expiration.js

```diff
@@ -165,6 +166,8 @@ class CacheExpiration {
           // Only return false if all the conditions are met.
           return false;
         }
+      } else {
+        this.expireEntries({cacheName, now});
```

Comment:
It's a little hard to line up the braces, but this `else` is associated with `if (dateHeader)`. I'll clarify in a comment what's going on.